locale/pt-br/site.json

```diff
@@ -0,0 +1,152 @@
+    },
+    {
+      "link": "feed/releases.xml",
+      "text": "Blog Node.js: Versões"
+    },
+    {
+      "link": "feed/vulnerability.xml",
+      "text": "Blog Node.js: Informações de Vulnerabilidade"
+    }
+  ],
+  "home": {
+    "text": "Início"
+  },
+  "about": {
+    "link": "about",
+    "text": "Sobre",
+    "governance": {
+      "link": "about/governance",
+      "text": "Projeto Governance"
+    },
+    "community": {
+      "link": "about/community",
+      "text": "Comunidade"
+    },
+    "workinggroups": {
+      "link": "about/working-groups",
+      "text": "Grupos de Trabalho"
+    },
+    "releases": {
+      "link": "about/releases",
+      "text": "Versões"
+    },
+    "resources": {
+      "link": "about/resources",
+      "text": "Recursos"
+    },
+    "trademark": {
+      "link": "about/trademark",
+      "text": "Marca"
+    },
+    "privacy": {
+      "link": "about/privacy",
+      "text": "Política de Privacidade"
+    }
+  },
+  "download": {
+    "link": "download",
+    "text": "Download",
+    "releases": {
+      "link": "download/releases",
+      "text": "Versões Anteriores"
+    },
+    "package-manager": {
+      "link": "download/package-manager",
+      "text": "Instale Node.js usando gerenciador de pacotes."
+    },
+    "shasums": {
+      "link": "SHASUMS256.txt.asc",
+      "text": "SHASUMS assinados para arquivos de versões",
+      "verify-link": "https://github.com/nodejs/node#verifying-binaries",
+      "verify-text": "Como verificar"
+    },
+    "install-on-linux": {
+      "text": "Instalando Node.js via arquivo binário no Linux."
+    }
+  },
+  "docs": {
+    "link": "docs",
+    "text": "Documentação",
+    "es6": {
+      "link": "docs/es6",
+      "text": "ES6 e além"
+    },
+    "api-lts": {
+      "link": "/dist/latest-%ver-major%/docs/api",
+      "subtext": "LTS",
+      "text": "%ver% API"
+    },
+    "api-current": {
+      "link": "/dist/latest-%ver-major%/docs/api",
+      "text": "%ver% API"
+    },
+    "guides": {
+      "link": "docs/guides",
+      "text": "Guias"
+    },
+    "dependencies": {
+      "link": "docs/meta/topics/dependencies",
+      "text": "Dependências"
+    }
+  },
+  "getinvolved": {
+    "link": "get-involved",
+    "text": "Participe",
+    "code-and-learn": {
+      "link": "get-involved/code-and-learn",
+      "text": "Programe + Aprende"
+    },
+    "collab-summit": {
+      "link": "get-involved/collab-summit",
+      "text": "Collab Summit"
+    },
+    "contribute": {
+      "link": "get-involved/contribute",
+      "text": "Contribue"
+    },
+    "conduct": {
+      "link": "https://github.com/nodejs/node/blob/master/doc/guides/contributing/coc.md#code-of-conduct",
+      "text": "Código de Conduta"
+    }
+  },
+  "security": {
+    "link": "security",
+    "text": "Segurança"
+  },
+  "blog": {
+    "link": "blog",
+    "text": "Novidades"
+  },
+  "foundation": {
+    "link": "https://foundation.nodejs.org/",
+    "text": "Foundation"
```

Comment:
```suggestion
    "text": "Fundação"
```

Reply:
@rogeriopvl dificilmente a gente fala aqui no brasil: Fundação Node.js. Acho q manter nesse caso em inglês é conveniente. O que você?
@Drieger oq tu achas?

Reply:
@glaucia86 acho que não teria problema. Tem alguns artigos e sites (não oficiais) escritos em português que utilizam de forma traduziada.

https://braziljs.org/blog/o-que-esperar-do-node-js-v7/
https://ecommercenews.com.br/noticias/parcerias-comerciais/godaddy-apoia-fundacao-node-js/

Dois exemplos de onde o termo foi utilizado. Dei uma olhada no site da mozilla também e lá eles fizeram essa tradução `Mozilla Foundation` -> `Fundação Mozilla` https://www.mozilla.org/pt-BR/about/

Se depois for levantado algum problema acredito que não terá nenhum impedimento em desfazer essa alteração.